kong/kong.lua

```diff
@@ -1,4 +1,4 @@
--- Kong, the biggest ape in town
+-- Kong, the biggest ape in town 1
```

Comment:
you can manually trigger a CI service from the repo without committing:

![image](https://cloud.githubusercontent.com/assets/183195/11667275/b4a5374e-9dbe-11e5-9772-8ba10c02e482.png)